models/backbone/vgg.py

```diff
@@ -0,0 +1,172 @@
+import torch
+import logging
+import torch.nn as nn
+
+from ..utils import conv3x3_group, norm_layer, layers, \
+     kaiming_init, constant_init, load_checkpoint
+
+
+def _make_res_layer(block,
+                    inplanes,
+                    planes,
+                    blocks,
+                    stride=1,
+                    dilation=1,
+                    use_gn=False):
+    layers = []
+    layers.append(
+        block(inplanes,
+              planes,
+              stride=stride,
+              dilation=dilation,
+              use_gn=use_gn,)
+    )
+    inplanes = planes
+    for i in range(1, blocks):
+        layers.append(
+            block(inplanes,
+                  planes,
+                  stride=1,
+                  dilation=dilation,
+                  use_gn=use_gn))
+    return nn.Sequential(*layers)
+
+
+class VGG(nn.Module):
+
+    """
+    vgg backbone.
+    paper: https://arxiv.org/pdf/1409.1556.pdf
+
+    Args:
+        vgg_name (string):vgg names,  distinguished by depth of vgg, from {11,
+        13, 16, 19}.
+        num_stages (int): vgg stages, normally 5.
+        strides (Sequence[int]): Strides of the first block of each stage.
+        dilations (Sequence[int]): Dilation of each stage.
+        out_indices (Sequence[int]): Output from which stages.
+        frozen_stages (int): Stages to be frozen (all param fixed). -1 means
+            not freezing any parameters.
+        use_gn (bool): Whether to use GN for normalization layers
+        bn_eval (bool): Whether to set BN layers to eval mode, namely, freeze
+            running stats (mean and var), only used when using BN.
+        bn_frozen (bool): Whether to freeze weight and bias of BN layers, only
+            used when using BN.
+    """
+
+    arch_settings = {
+        'vgg11': (layers.ConvModule, (1, 1, 2, 2, 2)),
+        'vgg13': (layers.ConvModule, (2, 2, 2, 2, 2)),
+        'vgg16': (layers.ConvModule, (2, 2, 3, 3, 3)),
+        'vgg19': (layers.ConvModule, (2, 2, 4, 4, 4)),
+    }
+
+    def __init__(self,
+                 vgg_name,
```

Comment:
modify this parameter to `depth`, and type is int, such as in `ResNet`, and update the docstring